projects/s2opc/build.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -eu
```

Comment:
This should be moved into upstream repo in future, see https://github.com/google/oss-fuzz/blob/master/docs/ideal_integration.md

Reply:
Hi,

I don't quite understand what you mean by moving this upstream, could you tell us more on that? 

Thanks,